forklift/LocatorPallet.py

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+'''
+LocatorPallet.py
+A module that contains a pallet definition for data to support the web api locator services and
+methods to keep them current.
+
+Pre-requisites
+    - The `secrets.py` file has been populated from the template file
+    - The locators in `self.services` have been created
+    - The locators are published to arcgis server
+    - The locators need to be published from the `shipTo` folder in the `secrets.py` file and that folder
+        needs to be registered as a data store in arcgis server
+
+Creating the locators
+    - Make sure your `Dev` secrets are populated as that configuration will be used
+    - In arcgis pro python execute `LocatorsPallet.py`
+'''
+
+from glob import iglob
```

Comment:
**chore**: remove dependency

Reply:
sorry, but the path.glob comment didn't post. So with that refactor it'd be unnecessary.